### ISSUE TITLE
Error: Cannot find module 'rxjs'

### ISSUE THREAD

**Issue by felixfbecker** [cartant/rxjs-tslint-rules#19]
We have a shared TSLint config that is used company-wide in all TS projects through an npm package. It includes rxjs-tslint-rules as a dependency and enables the rules. When using it on a project that does not use rxjs, linting errors with this exception:
```
{ Error: Cannot find module 'rxjs' from '/home/travis/build/felixfbecker/cli-highlight/node_modules/rxjs-tslint-rules/dist/support'
    at Function.module.exports [as sync] (/home/travis/build/felixfbecker/cli-highlight/node_modules/resolve/lib/sync.js:40:15)
    at read (/home/travis/build/felixfbecker/cli-highlight/node_modules/rxjs-tslint-rules/dist/support/knowns.js:9:25)
    at Object.<anonymous> (/home/travis/build/felixfbecker/cli-highlight/node_modules/rxjs-tslint-rules/dist/support/knowns.js:24:28)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18) code: 'MODULE_NOT_FOUND' }
```

**Comment by cartant** [comment]
That should be easy to fix. The implementation attempts to resolve the `rxjs` module to its `node_modules` location so that it can determine the RxJS version and available operators, etc. I'll publish a fix shortly. Thanks for raising the issue.

**Comment by cartant** [comment]
Should be fixed in 3.1.2.

**Comment by felixfbecker** [comment]
Thanks!
